src/audio/multiband_drc/multiband_drc.c

```diff
@@ -358,6 +385,31 @@ static int multiband_drc_cmd_set_data(struct comp_dev *dev,
 	return ret;
 }
 
+static int multiband_drc_cmd_set_value(struct comp_dev *dev,
+				       struct sof_ipc_ctrl_data *cdata)
+{
+	struct multiband_drc_comp_data *cd = comp_get_drvdata(dev);
+	int j;
+	int ret = 0;
+
+	switch (cdata->cmd) {
+	case SOF_CTRL_CMD_SWITCH:
+		comp_dbg(dev, "multiband_drc_cmd_set_value(), SOF_CTRL_CMD_SWITCH");
+		for (j = 0; j < cdata->num_elems; j++) {
+			cd->process_enabled = cdata->chanv[j].value;
```

Comment:
this will only grab the value the last of the switch, is this intended?

Reply:
Same as above. I will update this after I could build and run the switch.